compose-local-deps.yml

```diff
@@ -29,7 +29,14 @@ services:
   postgres:
     network_mode: bridge
     container_name: postgres
-    image: isaac-pg
+    image: postgres:9.6
+    volumes:
+      - pg-local:/var/lib/postgresql/data
+      - ./src/main/resources/db_scripts/postgres-rutherford-create-script.sql:/docker-entrypoint-initdb.d/00-isaac-create.sql:ro
+      - ./src/main/resources/db_scripts/postgres-rutherford-functions.sql:/docker-entrypoint-initdb.d/01-isaac-functions.sql:ro
```

Comment:
This is precisely the reason why using a custom built image is preferable in this case.